### DISCUSSION TITLE
request.body() or .json() etc requires a rogue argument

### DISCUSSION THREAD

**Discussion by kijetasantakalu** [encode/starlette#1743] [Q&A] [answered]
If one tries to get the json or body of a request, as seen in [this](https://www.starlette.io/requests/) docs page, it asks for a "self" argument which should not be asked.
![image](https://user-images.githubusercontent.com/73711897/177993371-9891c10f-4201-42eb-8064-8448944fa085.png)

**Comment by Kludex** [comment, marked as answer]
Closing this as I need an MRE, and it's outdated.
